docs/nddata/subclassing.rst

```diff
@@ -156,16 +156,16 @@ on the class or the instance::
     >>> ndd = NDDataPower([1,2,3])
 
     >>> # using it on the instance with one operand
-    >>> ndd.pow(3)
+    >>> ndd.pow(3)  # doctest: +SKIP
```

Comment:
What about this one?
```suggestion
    >>> ndd.pow(3)  # doctest: +ELLIPSIS
```

Reply:
Sorry, I did the last rebase without pulling the committed change from your suggestion. I won't do that this time.